### ISSUE TITLE
Bottom bar partially or totally hidden on 2021.2

### ISSUE THREAD

**Issue by vncastanheira** [LogicalError/realtime-CSG-for-unity#327]
![image](https://user-images.githubusercontent.com/1210284/143517551-08173f50-3857-4102-b38a-8d4b650b0846.png)

Steps to reproduce:

- Create a new 2021.2.3f1 project
- Import RealtimeCSG from the Asset Store
- Open the RealtimeCSG GUI

Tried to resize but it's still hidden.
If you place the Tools bar on the bottom, it's gonna be completely hidden. 

![image](https://user-images.githubusercontent.com/1210284/143517617-ffb6f49c-5cb6-4957-a854-783f3534b5a5.png)

**Comment by mifozski** [comment]
As a quick workaround, you can change hardcoded toolbar offset at `Assets/Plugins/RealtimeCSG/Editor/Scripts/View/GUI/SceneViewBottomBarGUI/SceneViewBottomBar.GUI.cs#27` from `+18` to `+27`:
```
bottomBarRect = new Rect(0, height - (CSG_GUIStyleUtility.BottomToolBarHeight + 27), 
```

**Comment by vncastanheira** [comment]
I'm looking for a way to detect if the toolbar is docked on the bottom.

**Comment by vncastanheira** [comment]
> As a quick workaround, you can change hardcoded toolbar offset at `Assets/Plugins/RealtimeCSG/Editor/Scripts/View/GUI/SceneViewBottomBarGUI/SceneViewBottomBar.GUI.cs#27` from `+18` to `+27`:
> 
> ```
> bottomBarRect = new Rect(0, height - (CSG_GUIStyleUtility.BottomToolBarHeight + 27), 
> ```
Yeah this works nicely, it must have something to do with the recent changes on the docked toolbar.

**Comment by Kerfuffles** [comment]
This issue should be fixed with #330, when it is ready to be merged with master.
